src/components/about/Philanthropy.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Image from "@/public/about/heart.svg";
```

Comment:
```suggestion
import Image from "@/public/about/heart.svg";
```
Rename this to something better than Image like Heart or something